pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -290,6 +294,11 @@ def transfer_distrib_spawn_state_on_fit_end(self, results):
             self.mp_queue.put(best_model_path)
             self.mp_queue.put(last_path)
             self.mp_queue.put(results)
+            self.mp_queue.put({
+                "callback_metrics": apply_to_collection(
+                    self.lightning_module.trainer.logger_connector.callback_metrics, torch.Tensor, lambda x: x.item()
+                ) or {}
```

Comment:
Why do we need `or {}`? Should return an empty dict by default if I'm not mistaken